### DISCUSSION TITLE
2013 05 monthly record

### DISCUSSION THREAD

**Discussion by buxuku** [buxuku/buxuku.github.io#135] [Announcements]
> [!NOTE]
> 本文通过 `mdx` 编辑，部分效果可能无法在纯 `markdown` 下渲染，更完整的阅读体验，欢迎访问博客页面 [2013 05 monthly record](https://blog.linxiaodong.com/blog/2013-05-monthly-record)

## 一。关于健康
5月份刚过了五一，就把岳母接上来安排在华西医院进行住院治疗，这已经是第二次来华西医院，至去年十月份第一次来华西医院已经过去了大半年，而从最初发病开始，已经整整过去了八个月之久，其间去过好几个医院治疗，中药，西药，输液，无数的治疗方案都均无效果。这次是实在没办法再这样拖下去了，于是决定到华西进行住院治疗。
当初怀疑过肺脓肿，也怀疑过肺结核，甚至怀疑过肺癌，但这一切均无法确症。5月份在华西住院一个月，也是各种检查无数，而检查结果无一例外的都是各种怀疑，依然无法确症。直到最近胸外科的医师过来看完各种报告，认为是炎症，决定进行手术，而接下来还要进行各种检查，等待手术。

与此同时，当时天天也带着小悦悦跟着跑上来，陪着岳母转了一天的医院，反而把小悦悦弄得高烧不退，吃药无效，最后只得输液，那一刻，我的心是最痛的，一个不到半岁的稚嫩小生命，却第一次扎着脑袋输着那可怕的液体，在我的记忆中，好像小孩子是千万不能输液的，我真想阻止这一切，而我却无能为力。源于成都这边的各种不便，小悦悦好了后，天天又不得不带着她回到老家，而在老家，又是一会儿这儿不好，一会儿又那儿不好。

有生以来，我已经两次在病危通知书上签下了字，第一次是09年爸爸病危的时候，第二次就是岳母这次，整个5月份，我是经常往医院跑，健康两个字，又再一次深深抨击着我的思考，虽然我一直知道健康很重要，可是我真的在健康上面投资了多少呢？我又为父母他们的健康付出了多少呢？去年和天天一起做了一次健康体检后，我们约定每年我们都要带着全家做一次健康体检，然而今年已经过去了一半，我们自己都还没有去做健康体验。今年年初我给自己定下了几个目标，好几个我都已经开始在执行了，而唯有最重要的健康计划却一直没有实施起来。这，是时候思考健康投资与计划的问题了。

## 二。关于梦想
这是5月份最简单的词汇，但也是一直令我深入思考的一个词汇，我们每天从早到晚，从年初忙碌到年尾，一生辛劳，而我们这一切是为了什么呢？大家都可能会说为了以后过上更好的生活，可什么样的生活才是更好的生活呢？我们有过一个概念吗？我们来到这个五彩缤纷的世界上，可我们的人生色彩是什么呢？我们活着一生，目的是为自己留下什么呢？我们的人生到底是在追求着什么呢？这一切问题，也许大多数人都有了自己的答案，而我却发现我无法回答我自己。倘若让我用文字，用彩笔描绘出我未来的梦想，我真的还是不知所措。

也许以前有很多天真的遐想，现在仔细想一起，还真的有着那么一份天真劲，我也想出去旅游，但我知道哪些地方有自己喜欢的风景吗?这个世界哪些地方我应该去呢？我又对那些地点了解多少呢？我也想来一场骑行冒险，但我的路线是什么呢？我的安全知识又储备在哪里呢？倘若说这些我都还可以去规划，去学习，但除了这些，我这一生又还有哪些梦想可以去追求呢？

好像人生就是这样慢慢地由童年的天真无瑕到成年的现实麻木，昨天，我们还可以三五成群地一起聊着对未来的无数梦想，而今天，大家都开始被这个现实的生活埋没了自己的激情与梦想，工作，家庭，孩子似乎就成了我们人生的全部。而我们的人生不是应该可以更精彩一些吗？我又将怎样才能重新拾回造梦的能力呢？

## 三。关于财富：富中之富
在曾经的日志里面，我写到了目标与目的，我们在为着目标奋斗的同时，我们不能忘了我们所追求的目的。而我也一直在思考，正如我认为健康，快乐是我们人生中最重要的，那么我们人生中到底还有哪些是最重要的呢？除了我们各自追寻的梦想不一样，而我们人生中有那些还是我们最应该追求的呢？同样是林伟贤的一堂课堂，让我对这个疑问有了答案，因为我是非常认可这样的格局，可以说他把我曾经的以及未来的追求很完整地做了一次归纳。
![](http://7te946.com1.z0.glb.clouddn.com/16-3-25/41000279.jpg)
什么是财富，不是我们这一生挣了多少钱，买了多少大的房子，拥有多少豪华的车，而只有我们在工作，学习，理财，健康，心灵，休闲，人脉，家庭这八大方面获得成功，我们才能算说是拥有了真正的富中之富。而我们的大多数人，却是把自己的几乎全部精力放在了工作之上，却忽视了其它几个方面的投入，所以纵然有千万金钱，那也不能算是财富。

我也开始思考自己自己，除了工作，我也仅仅是在学习，理财，心灵方面有所付出，而对健康，我却是一直没有执行，对于休闲，真的很想出去看看这个世界，去年的旅行计划，到今年都还没有开始，对于人脉的建立，确实应该开始努力了，三十岁年靠经验挣钱，三十岁后靠人脉挣钱，我不希望等到了三十岁，我才开始建立人脉，对于家庭，我认为我对父母们的关爱还远远不够，虽然与长辈们缺少交流的话语，但这不应该成为了可以少打电话，少慰问他们的理由。对于家这个摇篮，我不应该忽视太多。

## 四，关于记账
回首自己的记账之路，应该可以说从去年六月算起，在此期间，我也一直保持良好的记账习惯，在此期间，记账这个东西让我收益最大的就是能够清晰地知道自己每一个账户资金情况，因为我的账户特别多，每天的流水特别多，即使两三天不记账，我的账户就会乱得毫无头绪，所以，也正是有了这各客观因素，使得我一直很好地把记账坚持下来了。

但现在发现以前自己好像是为了记账而记账，每天只是把自己的流水记下来而已，虽然两个月前开始做预算，但也没有认真地去分析它。而记账的真正目的是什么？是为了分析自己的收入支出结构，制定财务目标，分析哪些支出为不必要的开支，哪些地方可以开源节流，以便控制自己的消费行为。真正引起我重视这一点是五月份的支出报表，虽然我知道我五月份的支出相对要多一些，但却没想到一下子飙升到了这么多，五月份总共支出高达一万九百多。这也不得不让我开始认真分析自己的支出情况，开始优化自己的支出结构，开始控制自己的消费习惯。六月份我将开始学习

认真分析账单，相信六月份我会在这方面有很大的进步。期待加油。
![](http://7te946.com1.z0.glb.clouddn.com/16-3-25/68665750.jpg)

## 五。关于阅读
五月份阅读完了刘克亚的《超高价营销》，他的上一本书《打造赚钱的机器》还没有看完就借给朋友去了，他的视频更多，但都没有认真地看完，所以这本书算是第一次认真地学习完的吧。确实是大师的理论体系，我相信我还得再认真地阅读好几遍才能够用自己的文字来总结出自己的学习体会。第一次阅读下来，让我对整个营销过程有了一个比较清晰的了解，包括营销前的打造自动系统，营销中的二合一促销模式，以及后续的专家，明星打造。这本书还需要继续再次阅读才能领略其中的精髓，同时也需要等待一个可以施展的舞台才能加以应用。

## 六。关于驾照
把“悲催”二字用在我考驾照这件事上一点也不为过。首先，我的身份证地址是成都的，因为我一直用的当时进大学后办的身份证，而自己的户口实际上已经迁回老家了，而考驾照必须要身份证上的地址和户口地址一致，无奈之下，在2月底回家重新办身份证，同时也办了一个有效期三个月的临时身份证，回来3月10号把资料交给教练，我原来有D照，而且是在德阳，本应该迁到成都来，但教练他说他会弄，我也就没管了。后来还兴冲冲地去驾校练过一次车。

交了资料后，教练什么也没有给我交代，所以等了一段时间后，到三月底才让我去体检，办完体检表后，又告之我上面的电话号码是要写另外一个电话号码的，无奈之下，又重新体检，完了后准备给他送过去，因为他回家要从我住那边过，所以他说他过来取，结果一直没有过来，这一等又是两周过去了，好不容易算是把体检表交给他了。

接下来又是等吧，这等到了五月几号，被告之得去把我的驾照迁到成都来，抽了一个空，跑去第四分局，结果搬走了，匆匆赶到新地点，被告之又要体检表，无奈，又得去体检，这次好像都跟我做对似的，中铁二局，金牛区人民医院都不给我盖章，因为我左眼的矫正视力不达标，去妇幼儿童医院，直接给我录系统，结果告之异地迁入数量超过限制，叫过一段时间。好吧，感觉妇幼儿童医院应该没问题，过了一段时间又去，结果被告之没有网络，无法联网，好吧，我又等。第三次去的时候，终于给办了下来，那个激动呀。然后赶紧跑去车管所把驾照给迁了。再一次把资料交给教练，再一次等待报名。

什么叫好事多磨，过了一周，教练又说驾照不行，是行政区域不对，连他都不明白的问题，5月28号，也就是我的临时身份证最后期限，赶紧又去车管所，重新把驾照弄好，原来是上次换证的时候，他们在录资料时把我行政区域选错了。然后领导过来赶紧把我的资料拿上去，希望能在今天把名报上。等了半天，结果又被告之：驾照显示异地迁入，要下个月3号后才能报名。好吧，我马上就是一个没有身份的人了，我只有再一次等待我的新身份证。

幸好我不是急着拿驾照，虽然整个过程比较曲折，但我一点没有抱怨教练的意思，同时我也非常感谢他一直都很友好地对待我，帮我想办法，亲自来拿资料，开车送我等等。再次感谢小燕给我介绍这样一个好教练。

**Comment by QWQanswer** [comment]
博主的文章引起了我的思考
